### PR TITLE
xena.fetchLeverageTiers

### PR DESCRIPTION
**Pull request by samgermain**
```
Node.js: v14.17.0
CCXT v1.72.84
xena.fetchLeverageTiers (BTC/USD:USDC)
1 ms
{
  'BTC/USD:USDC': [
    {
      tier: 1,
      notionalCurrency: 'BTC',
      notionalFloor: 0,
      notionalCap: 10,
      maintenanceMarginRatio: 0.025,
      maxLeverage: 20,
      info: {
        maxVolume: '10',
        initialRate: '0.05',
        maintenanceRate: '0.025'
      }
    },
    {
      tier: 2,
      notionalCurrency: 'BTC',
      notionalFloor: 10,
      notionalCap: 20,
      maintenanceMarginRatio: 0.05,
      maxLeverage: 10,
      info: { maxVolume: '20', initialRate: '0.1', maintenanceRate: '0.05' }
    },
    {
      tier: 3,
      notionalCurrency: 'BTC',
      notionalFloor: 20,
      notionalCap: 30,
      maintenanceMarginRatio: 0.1,
      maxLeverage: 5,
      info: { maxVolume: '30', initialRate: '0.2', maintenanceRate: '0.1' }
    },
    {
      tier: 4,
      notionalCurrency: 'BTC',
      notionalFloor: 30,
      notionalCap: 40,
      maintenanceMarginRatio: 0.15,
      maxLeverage: 3.3333333333333335,
      info: { maxVolume: '40', initialRate: '0.3', maintenanceRate: '0.15' }
    },
    {
      tier: 5,
      notionalCurrency: 'BTC',
      notionalFloor: 40,
      notionalCap: 60,
      maintenanceMarginRatio: 0.2,
      maxLeverage: 2.5,
      info: { maxVolume: '60', initialRate: '0.4', maintenanceRate: '0.2' }
    },
    {
      tier: 6,
      notionalCurrency: 'BTC',
      notionalFloor: 60,
      notionalCap: 150,
      maintenanceMarginRatio: 0.25,
      maxLeverage: 2,
      info: { maxVolume: '150', initialRate: '0.5', maintenanceRate: '0.25' }
    },
    {
      tier: 7,
      notionalCurrency: 'BTC',
      notionalFloor: 150,
      notionalCap: 200,
      maintenanceMarginRatio: 0.5,
      maxLeverage: 1,
      info: { maxVolume: '200', initialRate: '1', maintenanceRate: '0.5' }
    }
  ]
}
```

```
xena.fetchLeverageTiers ()
1 ms
{
  'AAVE/USD:USDC': [
    {
...
    }
  ]
}
```

### CODE REVIEW

**Comment by samgermain** [comment]
I noticed on this PR, it looks like Xena has `base` and `quote` currencies opposite

**Comment by kroitor** [comment]
Same comment as here: https://github.com/ccxt/ccxt/pull/11942#issuecomment-1038194703

**Comment by samgermain** [comment]
@kroitor Do you what this is talking about?

```
> ccxt@1.72.98 check-python-syntax

> cd python && tox -e qa && cd ..

qa create: /home/travis/build/ccxt/ccxt/python/.tox/qa

No handlers could be found for logger "filelock"

ERROR: invocation failed (exit code 1), logfile: /home/travis/build/ccxt/ccxt/python/.tox/qa/log/qa-0.log

================================== log start ===================================

RuntimeError: failed to query /home/travis/virtualenv/python3.7.6/bin/python3 with code 1 err: 'Traceback (most recent call last):\n  File "/usr/local/lib/python2.7/dist-packages/virtualenv/discovery/py_info.py", line 523, in <module>\n    print(PythonInfo()._to_json())\n  File "/usr/local/lib/python2.7/dist-packages/virtualenv/discovery/py_info.py", line 86, in __init__\n    self.distutils_install = {u(k): u(v) for k, v in self._distutils_install().items()}\n  File "/usr/local/lib/python2.7/dist-packages/virtualenv/discovery/py_info.py", line 152, in _distutils_install\n    d = dist.Distribution({"script_args": "--no-user-cfg"})  # conf files not parsed so they do not hijack paths\n  File "/home/travis/virtualenv/python3.7.6/lib/python3.7/site-packages/setuptools/dist.py", line 462, in __init__\n    for k, v in attrs.items()\n  File "/home/travis/virtualenv/python3.7.6/lib/python3.7/site-packages/setuptools/_distutils/dist.py", line 293, in __init__\n    self.finalize_options()\n  File "/home/travis/virtualenv/python3.7.6/lib/python3.7/site-packages/setuptools/dist.py", line 850, in finalize_options\n    for ep in sorted(loaded, key=by_order):\n  File "/home/travis/virtualenv/python3.7.6/lib/python3.7/site-packages/setuptools/dist.py", line 849, in <lambda>\n    loaded = map(lambda e: e.load(), filtered)\n  File "/home/travis/virtualenv/python3.7.6/lib/python3.7/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 196, in load\n    return functools.reduce(getattr, attrs, module)\nAttributeError: type object \'Distribution\' has no attribute \'_finalize_feature_opts\'\n'

=================================== log end ====================================

ERROR: InvocationError for command /usr/bin/python -m virtualenv --system-site-packages --no-download --python /home/travis/virtualenv/python3.7.6/bin/python3 qa (exited with code 1)

___________________________________ summary ____________________________________

ERROR:   qa: InvocationError for command /usr/bin/python -m virtualenv --system-site-packages --no-download --python /home/travis/virtualenv/python3.7.6/bin/python3 qa (exited with code 1)
```

File: js/xena.js [hunk 1]
```diff
@@ -1686,6 +1687,128 @@ module.exports = class xena extends Exchange {
...
+                    for (let j = 0; j < rates.length; j++) {
+                        const tier = rates[j];
+                        const cap = this.safeNumber (tier, 'maxVolume');
+                        const initialRate = this.safeString (tier, 'initialRate');
+                        tiers.push ({
+                            'tier': this.sum (j, 1),
+                            'notionalCurrency': market['base'],
+                            'notionalFloor': floor,
+                            'notionalCap': cap,
+                            'maintenanceMarginRatio': this.safeNumber (tier, 'maintenanceRate'),
```
Comment (kroitor) [inline js/xena.js:1793]: Let's rename this to `maintenanceMarginRate` everywhere.

**Pull request merged**